gradle/mockito-core/reproducible-build.gradle

```diff
@@ -0,0 +1,67 @@
+import java.security.DigestInputStream
+import java.security.MessageDigest
+
+/*
+ * The `checkReproducibility` task compares two sets of checksums, generated
+ * in two independent builds, from non-javadoc output archives.
+ *
+ * This task ensures that a change has not regressed the reproducibility of
+ * the Mockito project and its subprojects.
+ */
+
+task checkReproducibility {
+    doLast {
+        def first = new File("checksums-1.txt").text.split('\n')
+        def second = new File("checksums-2.txt").text.split('\n')
+
+        assert first.sort() == second.sort()
+    }
+}
+
+task getChecksums1(type: GradleBuild) {
+    tasks = ['clean', 'build']
```

Comment:
This runs the tests as well, which is not what we want. We only want to package the jar, to not incur a high cost on our CI. Can we only package the jar and verify the checksums on that?

Reply:
Oh, ofc, we can run `assemble` instead of `build` which creates the artifacts but doesn't run the tests. My mistake!